tidy3d/plugins/microwave/custom_path_integrals.py

```diff
@@ -292,6 +296,11 @@ def plot(
         matplotlib.axes._subplots.Axes
             The supplied or created matplotlib axes.
         """
+        if transpose:
+            # Please remove this warning once someone has verified that `transpose=True` works.
+            warn_untested_argument(
+                cls_name=type(self).__name__, func_name="plot", arg="transpose", val="True"
+            )
```

Comment:
The changes I made to this file should be moved into a different PR which can be tested separately.

I added this warning because I honestly can't remember if I tested the plot functions in this plugin with `transpose=True`.

Try running the cells in the [CharacteristicImpedanceCalculator.ipynb](https://github.com/flexcompute/tidy3d-notebooks/blob/develop/CharacteristicImpedanceCalculator.ipynb) (with `transpose=True`) to see if they work as expected.